tv_apps/smart-system/js/sync_manager.js

```diff
@@ -402,6 +402,11 @@
       // to do it for a periodic sync.
       this.unregisterSyncRequest();
 
+      if (!navigator.onLine) {
```

Comment:
Yes, it is intended. These sync requests can be triggered automatically by the sync scheduler, so we shouldn't show any error to the user and simply go back to the normal `enabled` state. For manual sync requests we show an error when the server request times out.